axelrod/strategies/darwin.py

```diff
@@ -23,7 +23,14 @@ class Darwin(Player):
     """
 
     name = "Darwin"
-    memory_depth = float('inf')
+    behaviour = {
```

Comment:
Ah - just got that now. Seems ok for the moment, but I wonder how long it will be before we have extend the dimensions!